packages/studioCMS/src/db/tables.ts

```diff
@@ -12,7 +11,7 @@ export const Session = defineTable({
 
 export const User = defineTable({
 	columns: {
-		id: column.text({ primaryKey: true, default: randomUUID() }),
+		id: column.text({ primaryKey: true }),
```

Comment:
Curious as to why this change?

Reply:
When you push the table schema if the randomUUID is included it will try to set a NEW Default over the old default...  creating a NEW SCHEMA every single push.  This change keeps that from happening but still requires data to be entered when creating new users/posts/postdata/etc.

Reply:
To explain a bit better, at first this was SUPPOSED to act as a replacement for the previous method Elian had used called `sql`uuid()`` which no longer works... Instead now i have all the randomUUID's being generated inside the actual codebase and not here due to the issue described in my previous comment.  But since it generates and saves a default UUID that is different every time one is to push the Database schema, every DB schema change becomes breaking.  This change will fix that issue and allow us to do database schema changes without causing breaking changes.

Reply:
Right, ok yeah makes sense!